### ISSUE TITLE
Links in speaker info can't be reached via keyboard navigation

### ISSUE THREAD

**Issue by kaitlinnewson** [code4lib/2022.code4lib.org#150]
On the speakers page, the speaker info box may contain links to a talk or workshop. While this box can be opened with the keyboard, these links can't be navigated to via keyboard navigation because they have `tabindex="-1"` set in `speaker_box.html`.

**Comment by kaitlinnewson** [comment]
In this case, I think the tabindex should be `-1` when the speaker info box is closed, but removed it it's open.
